packages/components/src/__layout__/Workflow/v3/subcomponents/Header/components/Titles/Titles.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { HTMLAttributes, ReactNode } from "react"
+import { Heading } from "~components/Heading"
+import { VisuallyHidden } from "~components/VisuallyHidden"
+import { OverrideClassName } from "~components/types/OverrideClassName"
+import styles from "./Titles.module.scss"
+
+export type WorkflowTitlesProps = {
+  workflowName: string
+  stepName: string
+  status?: ReactNode
```

Comment:
What are you're thoughts on either update this to `statusTag` / `headingTag`? My main thinking here is that now this can take a React Node, we may need to provide more clarity in terms of what we are expecting. Could even be just a type description if we want to leave the props the same.

Reply:
yeah good call, i like `statusTag` the most because even though yes it's open to anything, we're still expecting a status tag there

Reply:
Brill, sound good to me